nodebuilder/tests/blob_test.go

```diff
@@ -0,0 +1,173 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/blob"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/nodebuilder/tests/swamp"
+)
+
+func TestBlobModuleGet(t *testing.T) {
```

Comment:
Let's simply all these tests and decrease the time it takes to run all of them. We can do it if we combine them and make a little hierarchy. We can have one root TestBlobModule which would setup swamp full, bridge, and light nodes, and then we can have subtests for each of them, testing each method and NotFound. 

It's basically the same with the difference that we tests multiple things over the same nodes, minimizing overall test run time, but also test the same nodes can handle multiple ops over time, which is also nice.